src/handlers/prioritize.rs

```diff
@@ -38,6 +38,17 @@ impl Handler for PrioritizeHandler {
                     if e.label.as_ref().expect("label").name == config.label {
                         // We need to take the exact same action in this case.
                         return Ok(Some(Prioritize::Start));
+                    } else {
+                        if let Ok(glob) = glob::Pattern::new(&config.priority_labels) {
```

Comment:
We should put an error in the console at least with `log::error!` here if this fails

Reply:
Done!.